IntegrationTests/tests_01_allocation_counters/test_01_resources/test_tiny_array_cow_append_contents_of.swift

```diff
@@ -0,0 +1,30 @@
+//===----------------------------------------------------------------------===//
+//
+// This source file is part of the SwiftCertificates open source project
+//
+// Copyright (c) 2023 Apple Inc. and the SwiftCertificates project authors
+// Licensed under Apache License v2.0
+//
+// See LICENSE.txt for license information
+// See CONTRIBUTORS.txt for the list of SwiftCertificates project authors
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+//===----------------------------------------------------------------------===//
+
+@_spi(IntegrationTests) import X509
+
+func run(identifier: String) {
+    measure(identifier: identifier) {
+        var tinyArray = TinyArray<Int>()
+        tinyArray.append(1)
+        tinyArray.append(2)
+        tinyArray.append(3)
+        tinyArray.append(4)
+        tinyArray.append(5)
+        tinyArray.append(6)
+        tinyArray.append(7)
+        tinyArray.append(8)
+        return tinyArray.count
+    }
```

Comment:
Typically we try to run the bodies of these in loops that iterate 1000 times, just to distinguish allocations from within the loop from allocations outside of them.

Reply:
Done in https://github.com/apple/swift-certificates/pull/98/commits/6a42c66519c533f3eb23fa8e6667176d6f60ae4b

Reply:
That's what we want to test though. We want to test that we don't trigger CoW so we want to see asymptotical allocation behaviour. If we would trigger CoW we would see ~1000 allocations.

Reply:
Yes, but we want to measure the asymptotic behaviour too. Hence multiplying that out by 1000. We want to know the actual number of allocations in the loop above the noise floor, which we get by multiplying by 1000.